### PR TITLE
Mock bitcoin tests

### PR DESCRIPTION
**Pull request by HelloKashif**
This replaces all blockchair api calls in tests with mock calls.

One test testing the offsets in the api was removed as it was testing the blockchair api implementation.

### CODE REVIEW

File: packages/xchain-bitcoin/__mocks__/block-chair.ts [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+type MockConfig = {
+  url?: string
+}
+
+export default {
+  restore: mock.restore,
+  init: () => {
+    //Mock getAddress
+    mock.onGet(/\/dashboards\/address\//).reply(function (config: MockConfig) {
+      const id: any = config.url?.split('/').pop()
```
Comment (veado) [inline packages/xchain-bitcoin/__mocks__/block-chair.ts:16]: Please avoid `any` here and at other places in the you code ...

File: packages/xchain-bitcoin/__mocks__/block-chair.ts [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+      return [200, { data: { [id]: (addresses as any)[id] } }]
```
Comment (veado) [inline packages/xchain-bitcoin/__mocks__/block-chair.ts:17]: Same here. Can't types of [blockchair-api-types.ts](https://github.com/xchainjs/xchainjs-lib/blob/8b6ad2eec2e4ebf142d3dc86667c167403024778/packages/xchain-bitcoin/src/types/blockchair-api-types.ts) used here?

File: packages/xchain-bitcoin/__mocks__/block-chair.ts [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    })
+
+    //Mock getRawTx
+    mock.onGet(/\/raw\/transaction\//).reply(function (config: MockConfig) {
+      const id: any = config.url?.split('/').pop()
+      const result: any = { data: { [id]: (rawTransactions as any)[id] } }
```
Comment (veado) [inline packages/xchain-bitcoin/__mocks__/block-chair.ts:23]: `any`

**Review by veado — CHANGES_REQUESTED** [review]
Mocks for using Blockchair are looking good! Just few `any` related things ...

File: packages/xchain-bitcoin/__mocks__/block-chair.ts (hunk 1 above)
Comment (HelloKashif) [reply packages/xchain-bitcoin/__mocks__/block-chair.ts]: Fixed

File: packages/xchain-bitcoin/__mocks__/block-chair.ts (hunk 2 above)
Comment (HelloKashif) [reply packages/xchain-bitcoin/__mocks__/block-chair.ts]: Fixed

File: packages/xchain-bitcoin/__mocks__/block-chair.ts (hunk 3 above)
Comment (HelloKashif) [reply packages/xchain-bitcoin/__mocks__/block-chair.ts]: Fixed

**Review by veado — APPROVED** [review]

**Pull request merged**